back/lib/participation_method/community_monitor_survey.rb

```diff
@@ -2,12 +2,14 @@
 
 module ParticipationMethod
   class CommunityMonitorSurvey < NativeSurvey
+    ALLOWED_EXTRA_FIELD_TYPES = %w[page sentiment_linear_scale]
```

Comment:
Moved these to constants so I could easily get to them in `custom_field.printable?`